jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -243,6 +243,11 @@ static PluginConfigurationProcessor processCommonConfiguration(
             .setExposedPorts(ExposedPortsParser.parse(rawConfiguration.getPorts()))
             .setLabels(rawConfiguration.getLabels())
             .setUser(rawConfiguration.getUser().orElse(null));
+    rawConfiguration
+        .getWorkingDirectory()
+        .ifPresent(
+            workingDirectory ->
+                jibContainerBuilder.setWorkingDirectory(AbsoluteUnixPath.get(workingDirectory)));
```

Comment:
We should probably be consistent as to the validation behavior of all the configurations that are `AbsoluteUnixPath` (`workingDirectory`, `appRoot`, `volumes` (#1278)) - currently for `appRoot`, an invalid absolute Unix-style path is thrown as `AppRootInvalidException`, but here it would be thrown as the `IllegalArgumentException`.

Reply:
Makes sense. Added `WorkingDirectoryInvalidException`, which follows the pattern of `AppRootInvalidException`. (However, I see `InvalidContainerVolumeException` in #1278 is a bit different.)

Reply:
Hmm, maybe the naming should start with `Invalid...` since in most exception names, the adjective comes first? (ie. `IllegalArgumentException`, `NullPointerException`, `UnsupportedOperationException`)